### ISSUE TITLE
Can't validate confirmed

### ISSUE THREAD

**Issue by webowodev** [ssi-anik/form-request#2]
Hi, i accidently found that `confirmed` validation doesn't work if the request body was invalid XD, it don't give error validation message and resulted 500 status:
### My test request
```php
    public function testCanRegisterNewUser()
    {
        $data = [
            'name' => $this->faker->name,
            'email' => $this->faker->email,
            'password' => 'secret',
            'password_confirmation' => 'secretsss',
            'refferal_code' => null,
        ];

        $response = $this->json('post', '/api/auth/register', $data);
        dump($response->response->getContent());
        $response->assertResponseStatus(200);
        $response->seeJsonStructure([
            'status' => 'success',
            'message' => 'User registered',
        ]);
    }
```

### Result
```bash
Testing started at 11.47 ...
PHPUnit 9.5.3 by Sebastian Bergmann and contributors.

"""
{\n
    "message": "Form couldn't be validated.",\n
    "exception": "Anik\\Form\\ValidationException",\n
    "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/anik/form-request/src/FormRequest.php",\n
    "line": 16,\n
    "trace": [\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/anik/form-request/src/FormRequestServiceProvider.php",\n
            "line": 17,\n
            "function": "validate",\n
            "class": "Anik\\Form\\FormRequest",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/illuminate/container/Container.php",\n
            "line": 1226,\n
            "function": "Anik\\Form\\{closure}",\n
            "class": "Anik\\Form\\FormRequestServiceProvider",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/illuminate/container/Container.php",\n
            "line": 1190,\n
            "function": "fireCallbackArray",\n
            "class": "Illuminate\\Container\\Container",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/illuminate/container/Container.php",\n
            "line": 1175,\n
            "function": "fireAfterResolvingCallbacks",\n
            "class": "Illuminate\\Container\\Container",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/illuminate/container/Container.php",\n
            "line": 734,\n
            "function": "fireResolvingCallbacks",\n
            "class": "Illuminate\\Container\\Container",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/illuminate/container/Container.php",\n
            "line": 652,\n
            "function": "resolve",\n
            "class": "Illuminate\\Container\\Container",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/laravel/lumen-framework/src/Application.php",\n
            "line": 300,\n
            "function": "make",\n
            "class": "Illuminate\\Container\\Container",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/illuminate/container/BoundMethod.php",\n
            "line": 173,\n
            "function": "make",\n
            "class": "Laravel\\Lumen\\Application",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/illuminate/container/BoundMethod.php",\n
            "line": 124,\n
            "function": "addDependencyForCallParameter",\n
            "class": "Illuminate\\Container\\BoundMethod",\n
            "type": "::"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/illuminate/container/BoundMethod.php",\n
            "line": 36,\n
            "function": "getMethodDependencies",\n
            "class": "Illuminate\\Container\\BoundMethod",\n
            "type": "::"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/illuminate/container/Util.php",\n
            "line": 40,\n
            "function": "Illuminate\\Container\\{closure}",\n
            "class": "Illuminate\\Container\\BoundMethod",\n
            "type": "::"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/illuminate/container/BoundMethod.php",\n
            "line": 93,\n
            "function": "unwrapIfClosure",\n
            "class": "Illuminate\\Container\\Util",\n
            "type": "::"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/illuminate/container/BoundMethod.php",\n
            "line": 37,\n
            "function": "callBoundMethod",\n
            "class": "Illuminate\\Container\\BoundMethod",\n
            "type": "::"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/illuminate/container/Container.php",\n
            "line": 611,\n
            "function": "call",\n
            "class": "Illuminate\\Container\\BoundMethod",\n
            "type": "::"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php",\n
            "line": 386,\n
            "function": "call",\n
            "class": "Illuminate\\Container\\Container",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php",\n
            "line": 352,\n
            "function": "callControllerCallable",\n
            "class": "Laravel\\Lumen\\Application",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php",\n
            "line": 326,\n
            "function": "callLumenController",\n
            "class": "Laravel\\Lumen\\Application",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php",\n
            "line": 279,\n
            "function": "callControllerAction",\n
            "class": "Laravel\\Lumen\\Application",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php",\n
            "line": 264,\n
            "function": "callActionOnArrayBasedRoute",\n
            "class": "Laravel\\Lumen\\Application",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php",\n
            "line": 166,\n
            "function": "handleFoundRoute",\n
            "class": "Laravel\\Lumen\\Application",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php",\n
            "line": 426,\n
            "function": "Laravel\\Lumen\\Concerns\\{closure}",\n
            "class": "Laravel\\Lumen\\Application",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php",\n
            "line": 172,\n
            "function": "sendThroughPipeline",\n
            "class": "Laravel\\Lumen\\Application",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php",\n
            "line": 92,\n
            "function": "dispatch",\n
            "class": "Laravel\\Lumen\\Application",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/laravel/lumen-framework/src/Testing/Concerns/MakesHttpRequests.php",\n
            "line": 361,\n
            "function": "handle",\n
            "class": "Laravel\\Lumen\\Application",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/laravel/lumen-framework/src/Testing/Concerns/MakesHttpRequests.php",\n
            "line": 49,\n
            "function": "call",\n
            "class": "Laravel\\Lumen\\Testing\\TestCase",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/tests/Feature/user/RegisterUserTest.php",\n
            "line": 32,\n
            "function": "json",\n
            "class": "Laravel\\Lumen\\Testing\\TestCase",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/phpunit/phpunit/src/Framework/TestCase.php",\n
            "line": 1526,\n
            "function": "testCanRegisterNewUser",\n
            "class": "Feature\\user\\RegisterUserTest",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/phpunit/phpunit/src/Framework/TestCase.php",\n
            "line": 1132,\n
            "function": "runTest",\n
            "class": "PHPUnit\\Framework\\TestCase",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/phpunit/phpunit/src/Framework/TestResult.php",\n
            "line": 722,\n
            "function": "runBare",\n
            "class": "PHPUnit\\Framework\\TestCase",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/phpunit/phpunit/src/Framework/TestCase.php",\n
            "line": 884,\n
            "function": "run",\n
            "class": "PHPUnit\\Framework\\TestResult",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/phpunit/phpunit/src/Framework/TestSuite.php",\n
            "line": 677,\n
            "function": "run",\n
            "class": "PHPUnit\\Framework\\TestCase",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/phpunit/phpunit/src/Framework/TestSuite.php",\n
            "line": 677,\n
            "function": "run",\n
            "class": "PHPUnit\\Framework\\TestSuite",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/phpunit/phpunit/src/TextUI/TestRunner.php",\n
            "line": 667,\n
            "function": "run",\n
            "class": "PHPUnit\\Framework\\TestSuite",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/phpunit/phpunit/src/TextUI/Command.php",\n
            "line": 143,\n
            "function": "run",\n
            "class": "PHPUnit\\TextUI\\TestRunner",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/phpunit/phpunit/src/TextUI/Command.php",\n
            "line": 96,\n
            "function": "run",\n
            "class": "PHPUnit\\TextUI\\Command",\n
            "type": "->"\n
        },\n
        {\n
            "file": "/Users/dimaswibowo/code/laravel/fitmyday-api/vendor/phpunit/phpunit/phpunit",\n
            "line": 61,\n
            "function": "main",\n
            "class": "PHPUnit\\TextUI\\Command",\n
            "type": "::"\n
        }\n
    ]\n
}
"""

Expected status code 200 but received 500.
Failed asserting that 200 is identical to 500.
```

**Comment by ssi-anik** [comment]
Sorry that I missed your issue. Yes, that's because when the form is invalid, it throws an error. The upcoming release will solve this issue hopefully.

**Comment by ssi-anik** [comment]
Your issue is solved in tag `4.0`

**Comment by webowodev** [comment]
thanks @ssi-anik 🙏🏻
